### ISSUE TITLE
This plugin is in conflict with Beaver Builder

### ISSUE THREAD

**Issue by pySilver** [wp-shortcake/shortcake#720]
While Shortcode UI is active - most BB elements which uses wysiwyg are not able to render wysiwyg correctly. As a result it becomes (wysiwyg) inaccessible.

**Comment by carasmo** [comment]
Yes. And the insert post element button above tinymce does not launch in both Beaver Builder and Elementor.
